test/extended/prometheus/prometheus.go

```diff
@@ -76,6 +78,27 @@ var _ = g.Describe("[Feature:Prometheus][Late] Alerts", func() {
 
 		e2e.Logf("Watchdog alert is firing")
 	})
+
+	g.It("shouldn't exeeced the 500 series limit of total series sent via telemetry from each cluster", func() {
+		if !hasPullSecret(oc.AdminKubeClient(), "cloud.openshift.com") {
+			e2e.Skipf("Telemetry is disabled")
+		}
+		oc.SetupProject()
+		ns := oc.Namespace()
+
+		execPod := exutil.CreateCentosExecPodOrFail(oc.AdminKubeClient(), ns, "execpod", nil)
+		defer func() { oc.AdminKubeClient().CoreV1().Pods(ns).Delete(execPod.Name, metav1.NewDeleteOptions(1)) }()
+
+		tests := map[string]bool{
+			// We want to limit the number of total series sent, the cluster:telemetry_selected_series:count
+			// rule contains the count of the all the series that are sent via telemetry.
+			`max_over_time(cluster:telemetry_selected_series:count[2h]) >= 500`: false,
+		}
+		runQueries(tests, oc, ns, execPod.Name, url, bearerToken)
+
+		e2e.Logf("Total number of series sent via telemetry is below the limit")
```

Comment:
could it be useful to log the actual number?

Reply:
scratch that, it would require another query...

Reply:
Yeah its easy to look at the Prometheus dump in the CI artefacts afterwards to find the actual number I think.

Reply:
:+1: